pkg/security/secl/eval/eval.go

```diff
@@ -184,12 +184,12 @@ func patternToRegexp(pattern string) (*regexp.Regexp, error) {
 	return regexp.Compile("^" + quoted + "$")
 }
 
-func nodeToEvaluator(obj interface{}, opts *Opts, state *state) (interface{}, interface{}, lexer.Position, error) {
+func nodeToEvaluator(obj interface{}, opts *Opts, state *state, model Model) (interface{}, interface{}, lexer.Position, error) {
```

Comment:
I would rather use a map[Field]Field passed as Opts member instead of an interface

Reply:
👍🏻